include/deal.II/base/tensor.h

```diff
@@ -456,7 +456,7 @@ class Tensor
    * @note This function can also be used in CUDA device code.
    */
   constexpr DEAL_II_CUDA_HOST_DEV
-  Tensor() = default;
+  Tensor();
```

Comment:
Can't we just have
```
DEAL_II_ALWAYS_INLINE Tensor() = default;
```
?

Reply:
I will try that tomorrow.

Reply:
It works - that's the situation when I really appreciate code review because "I should have realized that on my own" by I have not.